src/wiseagents/vectordb/wise_agent_vector_db.py

```diff
@@ -24,9 +26,12 @@ class Document(BaseModel):
 
 class WiseAgentVectorDB(yaml.YAMLObject):
     """Abstract class to define the interface for a WiseAgentVectorDB."""
-    yaml_tag = u'!WiseAgentVectorDB'
     yaml_loader = WiseAgentsLoader
 
+    def __init__(self):
```

Comment:
Shouldn't an init method like this also be added for wise_agent_graph_db.py?

Reply:
Yeah, I missed it - added now